### DISCUSSION TITLE
Missing characters in fnt and png

### DISCUSSION THREAD

**Discussion by soccerob** [soimy/msdf-bmfont-xml#82] [Q&A]
I tried running my ttf with a custom charset that includes characters for multiple languages, but when I view the output fnt file, it only has 97 of the 170 characters that I'm expecting (from my charset). 
My charset is: https://gist.github.com/soccerob/c509e967017c1acdf06c2aca5ce4344b

!  "#$%&'()*+,-./0123456789:;?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[]^_`abcdefghijklmnopqrstuvwxyz{|}~¡«»¿ÀÁÂÄÅÆÇÈÉÊËÌÍÎÏÑÒÓÔÖØÙÚÛÜßàáâäåæçèéêëìíîïñòóôöøùúûüÿŒœŸ–—’“”„€\‘<=>°·•ãÃºõ

**Comment by soimy** [comment]
I just tested and all works fine.
Please post your `msdf-bmfont-xml` version, platform and font you are using. Also make sure to save your charset file in `utf-8`

**Comment by soccerob** [comment]
My mistake, was txt file was not saved as UTF-8

**Comment by soimy** [comment]
Good to hear that. I should mention this charset file encoding in the README
